front/migrations/20240423_backfill_customerio.ts

```diff
@@ -20,19 +20,41 @@ const backfillCustomerIo = async (execute: boolean) => {
     if (execute) {
       await Promise.all(
         c.map((u) =>
-          Promise.all([
-            CustomerioServerSideTracking.backfillUser({
-              user: u,
-            }).catch((err) => {
+          (async () => {
```

Comment:
FWIW, map does not support async function. I'd simply comment out this migration as it was a one time thing.